keras/requirements.txt

```diff
@@ -1,5 +1,5 @@
 keras
 numpy<2.0.0  # TODO(nabe): Remove this constraint once this issue is resolved. https://github.com/numpy/numpy/issues/26710
 optuna
-tensorflow-cpu
+tensorflow-cpu<2.19.0
```

Comment:
Could you add a todo comment to remove the upper version constraint, and open an issue to resolve this?